### PR TITLE
fix: Add the React component to i18n scan target

### LINKED ISSUE #1828: Let's make the item used in the React component included in the automatic translation target
### Main idea

When running the `make i18n`, the items used in the web component are automatically added to the `<language>.json` file.
But, if the items are used in the react component, it doesn't work. So let's make the item used in the React component included in the automatic translation target.

### Alternative ideas

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Choi-Jiwon-38**
This PR resolves #1828

### CODE REVIEW

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM

File: resources/i18n/ko.json [hunk 1]
```diff
@@ -376,7 +376,8 @@
     "SFTPExtraNotification": "개수 제한이 있으므로 다 사용한 후에는 삭제하는 것이 좋습니다. 일정 시간 업로드를 하지 않는 세션은 자동으로 삭제될 수 있습니다.",
     "ConnectionNotice": "주의",
     "Readmore": "더 알아보기...",
-    "Readless": "내용 접기"
+    "Readless": "내용 접기",
+    "SessionType": "__NOT_TRANSLATED__"
```
Comment (agatha197) [inline resources/i18n/ko.json]: Can you translate this line?

File: resources/i18n/ko.json [hunk 2]
```diff
@@ -443,7 +444,8 @@
     "Reset": "초기화",
     "OptOut": "비활성화",
     "Activate": "활성화",
-    "Unset": "초기화"
+    "Unset": "초기화",
+    "OK": "__NOT_TRANSLATED__"
```
Comment (agatha197) [inline resources/i18n/ko.json]: this line too.

File: resources/i18n/ko.json [hunk 3]
```diff
@@ -622,7 +624,8 @@
     "KeypairDetail": "키페어 정보",
     "AdminCanOnlyRemoveTotp": "다른 사용자의 이중 인증은 해제만 가능합니다.",
     "KeySeccessfullyDeleted": "키페어가 정상적으로 삭제되었습니다.",
-    "SignoutSeccessfullyFinished": "사용자 계정이 성공적으로 삭제되었습니다."
+    "SignoutSeccessfullyFinished": "사용자 계정이 성공적으로 삭제되었습니다.",
+    "ModifyUserDetail": "__NOT_TRANSLATED__"
```
Comment (agatha197) [inline resources/i18n/ko.json]: Same as above.

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]
I checked it worked well. Thanks for the contribution. And can you translate some sentences?

**Comment by Choi-Jiwon-38** [comment]
- `ModifyUserDetail` --> "사용자 상세 정보 수정"
스크린샷 2023-08-09 14 12 57

- `OK` --> "확인"
스크린샷 2023-08-09 14 13 45

- `SessionType` --> "세션 타입"
Write  based on existing content that translated.

**Review by Sujin-Kim1 — APPROVED** [review]

**Pull request merged**